munki_rebrand.py

```diff
@@ -186,6 +186,12 @@ def main():
                    default=None,
                    help="Optional tag to download a specific release of munki "
                    "e.g. 'v2.8.2'. Leave blank for latest Github code")
+    p.add_argument('-s', '--sign-package', action='store',
+                   default=None,
+                   help="Optional tag to sign distribution package with a "
+                   "Developer ID Installer certificate from keychain. Provide "
+                   "the certificate's Common Name. Ex: "
+                   "'Developer ID Installer: Munki (U8PN57A5N2)'")
```

Comment:
largely happy with this, but the wording in the help should change here. "optional tag" in the previous option refers specifically to a github release tag, so this should say something like "optionally sign..." etc